front/gatsby/src/components/profileCreationTutorial.tsx

```diff
@@ -0,0 +1,299 @@
+// Copyright 2018-2019 @paritytech/nomidot authors & contributors
```

Comment:
Could you rename the file to have a capital first letter, and the same name as the component?